api/base.cfg

```diff
@@ -1,11 +1,11 @@
 [buildout]
 index = https://pypi.org/simple/
-parts = instance plonesite robot-server
+parts = instance robot-server
 versions = versions
 
 extensions =
     mr.developer
-# auto-checkout =
+auto-checkout = plone.restapi
```

Comment:
@sneridagh don't we need to check out a specific branch of plone.restapi here?

Reply:
Yes, I already amended it, back to released.